Asterisk.2013/Asterisk.NET/Manager/ManagerConnection.cs

```diff
@@ -80,7 +84,6 @@ namespace AsterNET.Manager
 	public delegate void QueueCallerAbandonEventHandler(object sender, Event.QueueCallerAbandonEvent e);
 	public delegate void ZapShowChannelsCompleteEventHandler(object sender, Event.ZapShowChannelsCompleteEvent e);
 	public delegate void ZapShowChannelsEventHandler(object sender, Event.ZapShowChannelsEvent e);
-	public delegate void ConnectionStateEventHandler(object sender, Event.ConnectionStateEvent e);

```

Comment:
Yes. Agree!  But I come from Asterisk-Java project and there we fire "Connect/Disconnect/shutdown" events independently.  :)